src/components/tag-list/tag-list.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <ul class="tag-list">
+    <li
+      v-for="item in items"
+      :key="item.id"
+      class="tag-list__item body-big"
+    >
+      <strong>{{ item.amount }}</strong> {{ item.label }}
+    </li>
+  </ul>
+</template>
+
+<script lang="ts" setup>
+  type Props = {
+    items: {
+      id: string
+      amount: number
+      label: string
+    }[]
+  }
+
+  defineProps<Props>()
+</script>
+
+<style>
+  .tag-list {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    gap: var(--spacing-smaller);
+  }
+
+  .tag-list__item {
+    padding: .2em 1em .1em 1em;
```

Comment:
My gut feeling says this should be `rem` or is there a specific reason for `em` here?